core/src/com/unciv/models/ruleset/unique/Countables.kt

```diff
@@ -234,6 +234,61 @@ enum class Countables(
             UniqueParameterType.CivFilter.getKnownValuesForAutocomplete(ruleset)
                 .map { text.fillPlaceholders(it) }.toSet()
     },
+
+    WorkedTilesCity("Worked [tileFilter] Tiles in this city") {
```

Comment:
Actually, you're right. Having a `[cityFilter]` would allow for combining the two, as `[cityFilter]` has a `in this city`